examples/gallery/lines/polygons.py

```diff
@@ -0,0 +1,63 @@
+"""
+Filled polygons
```

Comment:
Does it make sense to change the title to "Choropleth Map"?

Reply:
Then I suggest to also rename the file itself.

Reply:
Done!